core/app/modules/kit-library/assets/js/hooks/use-content-types.js

```diff
@@ -21,7 +21,7 @@ function fetchContentTypes() {
 		{
 			id: 'page',
 			label: __( 'Pages', 'elementor' ),
-			doc_types: [ 'wp-page' ],
+			doc_types: [ 'page' ],
```

Comment:
Use the post type instead of the doc type.

Reply:
Done